pkg/controllers/grafana-operator/controller.go

```diff
@@ -170,49 +173,39 @@ func RegisterWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	logger := r.logger.WithValues("req", req)
+	log := r.logger.WithValues("req", req)
 
 	result := func(r reconcileResult) (ctrl.Result, error) {
 		if r.err != nil {
-			logger.Error(r.err, "End Reconcile - creation or updation error")
+			log.Error(r.err, "Error Reconciling resources")
 		} else if r.Result.Requeue || r.Result.RequeueAfter != 0 {
-			logger.Info("End Reconcile - requeue after", "res", r.Result)
+			log.V(6).Info("Re-queueing after", "res", r.Result)
 		} else {
-			logger.Info("End Reconcile - no requeue")
+			log.V(6).Info("Successfully reconciled resources")
 		}
 		return r.Result, r.err
 	}
 
-	logger.Info("Start reconcile")
-	logger.Info("Reconciling Grafana Operator Namespace")
-	if res := r.reconcileNamespace(ctx); res.stop {
-		return result(res)
-	}
-
-	logger.Info("Reconciling Grafana Operator OperatorGroup")
-	if res := r.reconcileOperatorGroup(ctx); res.stop {
-		return result(res)
-	}
-
-	logger.Info("Reconciling Grafana Operator Subscription")
-	if res := r.reconcileSubscription(ctx); res.stop {
-		return result(res)
+	log.V(6).Info("Reconciling resources")
```

Comment:
I think we should log this to identify start of the reconcile loop

Reply:
On the one hand yes, this could be beneficial. But the way that we currently reconcile is by re-triggering the loop over and over until all resources are created or updated. We would therefore often have empty reconciliation loops which do not modify anything. This is most pronounced when the operator starts and reconciles all resources again. So we would end up with a lot of `Reconciling resources` messages which could would be hard to interpret. 

IMO the operator should emit an info log when something actually changes. Indicating what the operator is doing is for me a trace or debug level message.

Reply:
Agreed, it is indeed a trace log. Let's go ahead with this change.